it/controllers/employment/SingleEmploymentSummaryEOYControllerISpec.scala

```diff
@@ -69,9 +69,9 @@ class SingleEmploymentSummaryEOYControllerISpec extends IntegrationTest with Vie
     val doYouNeedAnother: String = "Do you need to add another employment?"
     val yesText: String = "Yes"
     val noText: String = "No"
-    val change: String = "Change"
-    val remove: String = "Remove"
     val name: String = "Maggie"
+    val change: String = s"Change Change employment information for $name"
+    val remove: String = s"Remove Remove employment information for $name"
```

Comment:
Change and remove seem to be there twice which looks odd

Reply:
same as above